atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/TracingKeyValueService.java

```diff
@@ -69,10 +69,10 @@ protected KeyValueService delegate() {
     }
 
     @Override
-    public void addGarbageCollectionSentinelValues(TableReference tableRef, Set<Cell> cells) {
+    public void addGarbageCollectionSentinelValues(TableReference tableRef, Iterable<Cell> cells) {
         //noinspection unused - try-with-resources closes trace
         try (CloseableTrace trace = startLocalTrace("addGarbageCollectionSentinelValues({}, {} cells)",
-                tableRef, cells.size())) {
+                tableRef, Iterables.size(cells))) {
```

Comment:
Are we always still typically calling with a Collection where size is O(1)? If not, might want to remove this from the trace info, especially if we're passing lazily transformed Iterable/Collection/List